### ISSUE TITLE
UndefVarError using PlotlyJS backend

### ISSUE THREAD

**Issue by paulduf** [JuliaPlots/Plots.jl#1793]
Hello

I'm new to Julia and I'm trying to plot offline using PlotlyJS module.

Basic plot gives me this error. Surely I'm missing some important setup part but I can't debug this (at least not without spending many hours)

```
UndefVarError: _use_remote not defined

Stacktrace:
 [1] _show(::Base.GenericIOBuffer{Array{UInt8,1}}, ::MIME{Symbol("text/html")}, ::Plots.Plot{Plots.PlotlyJSBackend}) at /home/pol/.julia/packages/Plots/Ufx0i/src/backends/plotlyjs.jl:54
 [2] show(::Base.GenericIOBuffer{Array{UInt8,1}}, ::MIME{Symbol("text/html")}, ::Plots.Plot{Plots.PlotlyJSBackend}) at /home/pol/.julia/packages/Plots/Ufx0i/src/output.jl:201
 [3] #sprint#325(::Nothing, ::Int64, ::Function, ::Function, ::MIME{Symbol("text/html")}, ::Vararg{Any,N} where N) at ./strings/io.jl:101
 [4] sprint(::Function, ::MIME{Symbol("text/html")}, ::Vararg{Any,N} where N) at ./strings/io.jl:97
 [5] display_dict(::Plots.Plot{Plots.PlotlyJSBackend}) at /home/pol/.julia/packages/Plots/Ufx0i/src/init.jl:76
 [6] #invokelatest#1 at ./essentials.jl:697 [inlined]
 [7] invokelatest at ./essentials.jl:696 [inlined]
 [8] execute_request(::ZMQ.Socket, ::IJulia.Msg) at /home/pol/.julia/packages/IJulia/0cLgR/src/execute_request.jl:256
 [9] #invokelatest#1 at ./essentials.jl:697 [inlined]
 [10] invokelatest at ./essentials.jl:696 [inlined]
 [11] eventloop(::ZMQ.Socket) at /home/pol/.julia/packages/IJulia/0cLgR/src/eventloop.jl:8
 [12] (::getfield(IJulia, Symbol("##12#15")))() at ./task.jl:259
```

**Comment by mkborregaard** [comment]
Could you post the code to reproduce the error? What packages are you `using` etc.

**Comment by pnavaro** [comment]
```julia
using Plots
plotlyjs()
plot(Plots.fakedata(50,5),w=3)
```

**Comment by fpmenninger** [comment]
Same behavior here.
Using Windows 7, Julia 1.0.2.
using code of @pnavaro above, receive same error of:
```
UndefVarError: _use_remote not defined
```
And if comment out in line 54 as @ederag above, receive same error of 
```
UndefVarError: JupyterPlot not defined 
```

**Comment by SimonDanisch** [comment]
I think you can do a match + replace a la:
`_use_remote -> !use_local_dependencies`

**Comment by juliohm** [comment]
@SimonDanisch I found this line:

```julia
    if isijulia() && !_use_remote[]
```

and replaced it with:

```julia
    if isijulia() && use_local_dependencies[]
```

Now I don't get an error, but no plot is shown. I have also replaced the other occurrences but I think they are unrelated.

**Comment by mkborregaard** [comment]
Closed in #1885
